File: Trionic.java
class Solution {
    public boolean isTrionic(int[] nums) {
    
        int i = 0;
    int n = nums.length;
    if(n<4)
    {
        return false;
    }

    while(i+1<n && nums[i] < nums[i+1])
    {
        i++;
    }

   
    if(i==0)
    {
        return false;
    }

    int  p = i;

    while(i+1<n && nums[i] > nums[i+1])
    {
        i++;
    }

    if(p==i)
    {
        return false;
    }

    int q = i;
   
    

    while(i+1<n && nums[i] < nums[i+1])
    {
        i++;
    }

    if(q==i)
    {
        return false;
    }

    return i==n-1;


    }
}

File: Trionic.py
size = int(input("Enter size"))
list = []
for i in range(size):
    list.append(int(input(i)))
    i+=1

print(list)


i = 0
n = len(list)
        

while i+1 < n and list[i] < list[i+1]:
    i+=1

        if i==0:
            return False

        p = i

        while i+1<n and nums[i] > nums[i+1]:
            i+=1

        if p==i:
            return False

        q = i

        while i+1<n and nums[i] < nums[i+1]:
            i+=1

        if q==i:
            return False

        return i==n-1